requirements.txt

```diff
@@ -0,0 +1 @@
+qiskit-terra>=0.24
```

Comment:
This seems like not enough. Algorithms has a direct dependence on `numpy` and `scipy` so these should be listed here, i.e  direct dependents and if there are others these need to be here too.